docroot/profiles/custom/cgov_site/themes/custom/cgov/src/styles/_managed-lists.scss

```diff
@@ -22,7 +22,7 @@
           + .container {
             margin: 0;
             width: auto;
-            padding: 0 mq-px2em(15px);
+            padding-right: mq-px2em(15px);
```

Comment:
This does not seem to be the correct fix as it is not specifically targeting accordion content.

A good example on the ODE content is https://ncigovcdode522.prod.acquia-sites.com/about-nci/organization/ccct